client/src/routes/CreateSessionModal/CreateSessionModal.tsx

```diff
@@ -295,6 +319,10 @@ const CreateSessionModal = () => {
   );
 
   const hasProfile = Boolean(user?.displayName) && Boolean(user?.photoURL);
+  const userProfile = {
+    displayName: user?.displayName ?? undefined,
+    photoURL: user?.photoURL ?? undefined,
```

Comment:
Do you really need the `??` here?

Reply:
Yeah, else typeScript complains it can be null. Or is there another way @gewfy ?